getting-started/installing.md

```diff
@@ -31,9 +30,9 @@ The easiest way to install Gleam on Linux, Windows, and Apple macOS is to downlo
 prebuilt version of the compiler from the [GitHub release
 page](https://github.com/gleam-lang/gleam/releases).
 
-### macOS
+### UNIX Cross-platform
```

Comment:
Brew isn't portable so this isn't quite right. I'm not sure asdf is either. Instead let's add brew to Linux.

Reply:
Done